Pillar_II/esm/src/dummy_prune_analysis.py

```diff
@@ -7,37 +11,40 @@
 from pycompss.api.parameter import IN  # type: ignore
 from pycompss.api.task import task  # type: ignore
 from pycompss.runtime.management.classes import Future  # type: ignore
-from hecuba import StorageDict
-import time
-import random
-
-class MetaDictClass(StorageDict):
-   '''
-   @TypeSpec dict <<keyname0:str>,valuename0:str>
-   '''
-
-def simulation_started(expid):
-    state = MetaDictClass.get_by_alias(expid) 
-    if len([k for k in s.keys()]) > 1:
-        return True
-    else:
-        return False
+
+"""eFlows4HPC ensemble members pruning."""
+
+CHECK_FOR_PRUNING_SLEEP_TIME_SECS = 5
+"""How long should we sleep for"""
+
+logger = logging.getLogger(__name__)
+
 
 @task(expid=IN)
-def esm_analysis_prune(expid):
-    
+def esm_analysis_prune(expid: str):
+    """This has to be called by PYCOMPSs as analysis.
+
+    Args:
+        expid: The experiment ID.
+    """
+    # N.B.: importing this results in a network query to Hecuba servers,
+    #       which fails if the servers are not available.
+    from hecuba import StorageDict  # type: ignore
```

Comment:
Importing this breaks the code execution if you run locally, on CICD, etc, as this Hecuba Python code tries to connect to Hecuba servers (IMHO it would be better if it only did that after a call to `.connect(...)` or something like that, but for now this should solve CICD issues.